debugger-guide.md

```diff
@@ -0,0 +1,96 @@
+
+# Debugger Guide 👨‍🔧
+
+Here we will learn how to use debugger for Keploy in VS Code.
+
+To bring up the **Run and Debug** view, select the **Run and Debug** icon in the **Activity Bar** on the side of VS Code. You can also use the keyboard shortcut `Cmd + Shift + D` for MacOS and `Ctrl + shift + D` for Windows.
+
+If running and debugging is not yet configured (no `launch.json` has been created), VS Code shows the Run start view.
+
+![config-debug](https://media.discordapp.net/attachments/1166595840505294908/1207236720647471134/image.png?ex=65dee9cf&is=65cc74cf&hm=4986ddca66cbc03ff7301ced02c819b7d82a1a3705584883271b1b2b9eb91a69&=&format=webp&quality=lossless)
+
+Click on **create a launch.json file**  👉  **Go**  👉  **Go: Launch Package**
+
+Navigate to `launch.json` to begin crafting JSON objects.
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Record",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "asRoot": true,
+            "console": "integratedTerminal",
+            "program": "main.go",
+            "args": [
+                "record",
+                "-c",
+                "<path_to_executable>"
+            ],
+        },
+        {
+            "name": "Test",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "asRoot": true,
+            "console": "integratedTerminal",
+            "program": "main.go",
+            "args": [
+                "test",
+                "-c",
+                "<path_to_executable>", // We can also put docker run command here.
```

Comment:
there is no point of using a docker run command here, as the breakpoints will not work. Have you tested it before adding the comment?